project/GenerateFixtures.scala

```diff
@@ -43,13 +43,16 @@ case class GenerateFixtures(fixturesDir: File, frontend: String, version: String
     def patchedVersionDoesNotExist(example: JsObject): Boolean =
       !(patchedFixturesDir / (example \ "exampleId").as[String]).isDirectory
 
+    def replaceUnwantedCharacters(text: String) = text.replaceAll("/", "-")
```

Comment:
We have a problem when generating example folders from the examples provided by the govuk library
some example names have special characters in them. In this case the one example contained a / in the name

`when neither maxlength/maxwords nor textarea description are set`

But I have seen a few example folders with other special characters like below

`radios-with-fieldset,-error-message-and-describedBy
notification-banner-role=alert-overridden-to-role=region,-with-type-as-success
checkboxes-with-single-option-(and-hint)-set-'aria-describedby'-on-input,-and-describedBy`

I will need another ticket to sort this issue out.